docs: expands design guidelines for bullseye layout. (#3664)
patternfly/patternfly-org: packages/documentation-site/patternfly-docs/content/design-guidelines/layouts/bullseye.md

id: Bullseye
section: layouts
---
The **bullseye** layout centers content, both vertically and horizontally within a container.

## Elements 

<img src="./img/bullseye-example.png"  alt="PatternFly website screenshot showing modal created with bullseye layout."  width="3330px"/>

1. **Bullseye:** The bullseye container, which contains the bullseye item.

> 2. **Bullseye item:** An individual section of content to be placed in the bullseye.

Author: ```suggestion
2. **Bullseye item:** An individual section of content to be placed in the bullseye container.
```

## Usage 

> A bullseye layout can only contain one section of content. For example,  bullseye layout is commonly used with [modal components](/components/modal), to overlay important content on top of another layout, like a gallery of cards.

Author: ```suggestion
A bullseye layout can only contain one section of content. For example, bullseye layout is commonly used with the [modal component](/components/modal) to overlay important content on top of another layout, like a gallery of cards.
```

> If you need to center more than one section of content on a page, you should choose a different PatternFly layout.

Author: ```suggestion
If you need to center more than one section of content on a page, you should choose a different [PatternFly layout](/layouts/about).
```

> If you need to center multiple sections of content on a page, you should choose a different PatternFly layout.

Author: ```suggestion
If you need to center more than one section of content on a page, you should choose a different PatternFly layout.
```
